gbfs-validator/cli.js

```diff
@@ -1,17 +1,65 @@
 const { inspect } = require('util')
+const commander = require('commander')
+const fs = require('fs')
+const fsPath = require('path')
 const GBFS = require('./')
+const pjson = require('./package.json');
 
-if (!process.argv[2]) {
-  console.error('Usage: gbfs-validator GBFS_URL')
-  process.exit(1)
+getFeedValidationReport = async (url) => {
+  const gbfs = new GBFS(url)
+  return gbfs.validation()
 }
 
-const gbfs = new GBFS(process.argv[2])
+parseOptions = () => {
+  commander
+    .version(pjson.version, '-v, --version')
```

Comment:
It's always `1.0.0` because we don't use it right now.
We return the git hash instead https://github.com/MobilityData/gbfs-validator/blob/master/gbfs-validator/gbfs.js#L3

Reply:
This is in preparation for publishing an npm package. In the case of the command line, if we would like to include the commit hash, more work is needed to make available the hash information on local computers.